elm_deps_sync.py

```diff
@@ -2,55 +2,39 @@
 from __future__ import print_function
 
 import sys
-from collections import OrderedDict
-import json
 import argparse
 
+import elm_package
```

Comment:
I was edging against this so that the file could be run by itself without needing the whole repo, but maybe it doesn't matter